amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSourceStage.scala

```diff
@@ -58,10 +58,6 @@ final class AmqpSourceStage(settings: AmqpSourceSettings, bufferSize: Int)
         // we have only one consumer per connection so global is ok
         channel.basicQos(bufferSize, true)
         val consumerCallback = getAsyncCallback(handleDelivery)
-        val shutdownCallback = getAsyncCallback[Option[ShutdownSignalException]] {
```

Comment:
Same as in the RPC stage.

I modify the exception messages to be consistent in both stages.

Also, `if (unackedMessages == 0) completeStage()` is wrong logic since once the channel has been shut down, no more messages can be acked so all that's left is failing.